internal/sync/sync.go

```diff
@@ -24,7 +24,11 @@ import (
 	"go.uber.org/zap"
 )
 
-var errIncompatibleStateRoot = errors.New("incompatible state root")
+var (
+	txnV0Id                  = new(felt.Felt).SetHex("0x0")
+	txnV1Id                  = new(felt.Felt).SetHex("0x1")
```

Comment:
Assignment using the helpers `SetOne` and `SetZero` respectively are less costly.